isos/bootstrap/bootstrap

```diff
@@ -59,21 +59,21 @@ if mount -t ext4 /dev/disk/by-label/containerfs ${MOUNTPOINT}; then
         cp /bin/tether ${MOUNTPOINT}/.tether/tether-bin
 
         # temporary hack for waiting the network
-        while ! ip addr show eno1 | grep " inet "; do sleep 1;done
```

Comment:
I believe that George will be adding a simple dhcp server to the tether to address this issue in the near future.